Android/wallet/src/main/java/com/flow/wallet/wallet/KeyWallet.kt

```diff
@@ -155,8 +172,7 @@ class KeyWallet(
                 val p256Accounts = async {
                     p256PublicKey?.let { publicKey ->
                         try {
-                            val encodedKey = BaseEncoding.base16().lowerCase().encode(publicKey)
-                            println("Looking up P256 accounts for key: $encodedKey on network $network")
+                            val encodedKey = publicKey.toFlowIndexerHex()
```

Comment:
It should be just data to hex, drop the first byte.
Refer to swift code:
https://github.com/onflow/Flow-Wallet-Kit/blob/daab7c3e1368738a77dbd3f2f45d2b0ef639534f/iOS/FlowWalletKit/Sources/Utils.swift#L84-L101
https://github.com/onflow/Flow-Wallet-Kit/blob/daab7c3e1368738a77dbd3f2f45d2b0ef639534f/iOS/FlowWalletKit/Sources/Keys/PrivateKey.swift#L208